src/Vault.sol

```diff
@@ -240,69 +234,46 @@ contract Vault {
 
     /**
      * @notice Can be called by authorised applications to increase a margin position.
-     * @param baseCurrency The Base-currency in which the margin position is denominated
+     * @param baseCurrency_ The Base-currency in which the margin position is denominated
      * @param amount The amount the position is increased.
      * @return success Boolean indicating if there is sufficient free margin to increase the margin position
      * @dev The Liquidation Threshold will automatically be updated on every increase of margin,
      * but not automatically on a decrease of margin (since this derisks the vault).
      */
-    function increaseMarginPosition(address baseCurrency, uint256 amount)
+    function increaseMarginPosition(address baseCurrency_, uint256 amount)
         public
+        view
         onlyAuthorized
         returns (bool success)
     {
-        if (baseCurrency != vault.baseCurrency) {
+        if (baseCurrency_ != baseCurrency) {
             return false;
```

Comment:
to check if this can be replaced by a revert

Reply:
As just discussed, we'll keep it as it is, since TrustedCreditor might implement custom logic to deal with it.